overlord/configstate/tasksets.go

```diff
@@ -35,15 +36,25 @@ func init() {
 	snapstate.Configure = Configure
 }
 
+func maxConfigureHookRuntime() time.Duration {
+	maxRuntime := 5 * time.Minute
+	if mrs := os.Getenv("SNAPD_CONFIGURE_HOOK_TIMEOUT"); mrs != "" {
+		if mr, err := time.ParseDuration(mrs); err == nil {
```

Comment:
I think its fine to ignore an error as we only use it in tests, but I can do it still :)